src/oc_erchef/schema/deploy/keys_update_trigger.sql

```diff
@@ -5,29 +5,35 @@
 BEGIN;
 
 CREATE OR REPLACE FUNCTION add_key() RETURNS TRIGGER AS $add_key$
-   BEGIN
-     IF NEW.public_key IS NOT NULL THEN
-       INSERT INTO keys
-         (id, key_name, public_key, key_version, created_at,  expires_at) VALUES
-         (NEW.id, 'default', NEW.public_key, NEW.pubkey_version, now(), 'infinity'::timestamp);
-     END IF;
-     RETURN NULL; -- result is ignored since this is an AFTER trigger
-   END;
+  BEGIN
+    IF NEW.public_key != 'thisisnotakey' THEN
```

Comment:
See the NULL comparision comment for create_and_update_users.sql above

Reply:
updated.